frontend/src/registration/components/CongratulationsPage/CongratulationsPage.tsx

```diff
@@ -45,6 +47,13 @@ export class CongratulationsPage extends React.PureComponent<Props, State> {
           <p>
             <WrappedMessage messages={messages} id="congratsMessage2" />
           </p>
+          <NavLink exact to={ROUTES.Console.HOME}>
```

Comment:
The `NavLink` component here is wrapping around the entire div, so clicking anywhere on the div will trigger the link.
This isn't a good accessibility practice, can you move the `div` outside the `Navlink`? 
![2020-05-04_09-03](https://user-images.githubusercontent.com/27893385/80963814-1c005600-8de6-11ea-951c-bf1cfeb41925.png)

Reply:
Oh yeah :open_mouth: 
I have missed that completely, thanks a lot for catching this :smiley: